nixos/tests/foliate.nix

```diff
@@ -0,0 +1,46 @@
+import ./make-test-python.nix ({ pkgs, ... }: {
+  name = "foliate";
+  meta = with pkgs.lib.maintainers; {
+    maintainers = [ kamadorueda ];
+  };
+
+  nodes.machine = { ... }: {
+    config.programs.foliate.enable = true;
+  };
+
+  testScript = ''
+    from textwrap import dedent
+
+    machine.start()
+    machine.wait_for_unit("dbus.service")
+    machine.succeed("foliate --version")
+
+    machine.copy_from_vm("/etc/dconf/profile/foliate")
+    with open(machine.out_dir / "foliate") as file:
+        content = file.read()
+        assert content == dedent("""
+            user-db:user
+            system-db:ibus
+        """)[1:], repr(content)
+
+    machine.copy_from_vm("/etc/dconf/db/foliate.d/00-nixos-settings")
+    with open(machine.out_dir / "00-nixos-settings") as file:
+        content = file.read()
+        assert content == dedent("""
+          [com/github/johnfactotum/Foliate]
```

Comment:
Please don't assert that exact code from above. This just makes it that we need to update the test way more often and it is not a valuable test.